### DISCUSSION TITLE
Brew upgrade is failing, who could help please?

### DISCUSSION THREAD

**Discussion by oneslash** [neovim/neovim#26841] [General]
Hi everyone, I use the brew version of neovim and I have noticed that the build is failing in the proposed Pull Request (https://github.com/Homebrew/homebrew-core/pull/158567) and I wasn't sure if this is the right place to ask for help.

Could anyone support in there please? 

Thank you.

**Comment by clason** [comment]
This is a homebrew issue, so any help must come from them.

**Reply by oneslash** [reply]
I am sorry then for the spam here.
